### DISCUSSION TITLE
snakebids init

### DISCUSSION THREAD

**Discussion by akhanf** [khanlab/snakebids#104] [Ideas]
Instead of generating a brand new workflow with snakebids-create, would be nice to have an "init" interface (ie inspired by `git init`, or `poetry init`, ...)  that you run to turn the current working directory workflow into a snakebids workflow, e.g. adding in the config files, run.py, github actions, pyproject etc.. Could still have an interactive prompt too..

I find it is more typical (at least for my use cases) to create a basic snakemake workflow first, then adopt snakebids, rather than initializing with snakebids from the start..

Thoughts?

**Comment by pvandyken** [comment]
I guess this would just add the `run.py` file in? If you wanted to populate a config, you'd probably have to commit to having two separate config.yaml files, i.e. one `config.yaml` and one `snakebids.yaml`.
